lib/tasks/eslint.rake

```diff
@@ -7,6 +7,7 @@ namespace :eslint do
   desc %{Run ESLint against the specified JavaScript file and report warnings (default is 'application')}
   task :run, [:filename] => :environment do |_, args|
     warnings = ESLintRails::Runner.new(args[:filename]).run
+    puts
```

Comment:
Oh, it looks like there's another in there, too. So, nevermind. This is a stray puts :)